tests/spec/core/dfn-index-spec.js

```diff
@@ -0,0 +1,244 @@
+"use strict";
+import { flushIframes, makeRSDoc, makeStandardOps } from "../SpecHelper.js";
+
+describe("Core — dfn-index", () => {
+  afterAll(flushIframes);
+
+  it("adds default heading to section#index", async () => {
+    const body = `<section id="index">
+      <p>PASS</p>
```

Comment:
this pass is not checked in the test... might be just worth checking that PASS is next sibling of the heading.